Assets/Scripts/Player/Shield.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using UnityEngine;
+
+namespace Player
+{
+    public class Shield : MonoBehaviour
+    {
+        private void OnTriggerEnter2D(Collider2D other)
+        {
+            if (other.gameObject.layer == 10)
```

Comment:
Please save a prefab of the player so that we can reproduce locally easier later onwards. Perhaps in `Assets/Actors`?

Reply:
Done.

Reply:
Looks like the scene has two players now. Otherwise looks good, thanks.